### PR TITLE
test: add remove_files tests

### PR DESCRIPTION
**Pull request by athith-g**
Adds tests for [remove_files](https://github.com/elixir-cloud-aai/protes-middleware-crypt4gh/blob/main/crypt4gh_middleware/decrypt.py#L94)

## Summary by Sourcery

This pull request introduces new tests for the `remove_files` function, ensuring its correct behavior in various scenarios including handling empty directories and non-existent directories.

- **Tests**:
    - Added tests for the `remove_files` function to ensure files in a directory are removed successfully.
    - Included tests to verify no errors are raised when an empty directory is passed to `remove_files`.
    - Added a test to check that a ValueError is raised when a non-existent directory is passed to `remove_files`.

### CODE REVIEW

File: tests/decryption/test_decrypt.py [hunk 1]
```diff
@@ -145,3 +147,32 @@ def test_permission_error(self, tmp_path):
...
+        assert all(file.exists() for file in files)
+        remove_files(tmp_path)
+        assert not any(file.exists() for file in files)
+
+    def test_empty_dir(self, tmp_path):
+        """Test that no error is raised when an empty directory is passed."""
+        empty_dir = tmp_path/"empty"
+        empty_dir.mkdir()
+        remove_files(empty_dir)
+        assert not any(empty_dir.iterdir())
```
Comment (jvkersch) [inline tests/decryption/test_decrypt.py:166]: Maybe make this a bit more explicit?
```suggestion
        assert len(list(empty_dir.iterdir())) == 0
```
(assuming the two are equivalent, of course)

**Review by jvkersch — APPROVED** [review]
LGTM, just one small comment.

**Review by uniqueg — APPROVED** [review]

File: tests/decryption/test_decrypt.py [hunk 2]
```diff
@@ -11,7 +11,8 @@
     get_private_keys,
     decrypt_files,
     move_files,
-    remove_files
+    remove_files,
+    get_args
```
Comment (uniqueg) [inline tests/decryption/test_decrypt.py:15]: When you "list" params/args like that, please always add a trailing comma to the last item as well. It makes the diff view cleaner when changed (as demonstrated here).

Also, please order in ascencing alphanumeric order.

Finally, it seems that you are adding a test for `get_args()`, not `remove_file()`, no? If so, please adjust the PR title accordingly.

**Review by uniqueg — CHANGES_REQUESTED** [review]
Tests are great, but please address the single comment :pray:

File: tests/decryption/test_decrypt.py (hunk 2 above)
Comment (athith-g) [reply tests/decryption/test_decrypt.py]: This PR is for `remove_file()` but I merged the `get_args()` tests into main first that's my bad.

**Review by uniqueg — APPROVED** [review]

**Pull request merged**